browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4124,6 +4124,12 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_SAFEBROWSING_ERRORS_PREF" desc="A preferences to show Safe Browsing errors">
       Show Safe Browsing errors
     </message>
+    <message name="IDS_INCOGNITO_SCREENSHOT_TITLE" desc="A title to show incognito screenshot">
+      Private tab screenshots
+    </message>
+    <message name="IDS_INCOGNITO_SCREENSHOT_SUMMARY" desc="A summary to show incognito screenshot">
+      Enables Private tab screenshots.
+    </message>
```

Comment:
I think we should stay away from using `incognito` here. Even if you prefer to keep it in the IDS_ name, we should definitely use `private` in the `desc`.

Reply:
hey @mkarolin i have updated above here : https://github.com/brave/brave-core/pull/28142/commits/b048b24b2527b2a58d7f38f859037017949562a4 based on suggestion : https://github.com/brave/reviews/issues/1889#issuecomment-2725388414